packages/editor/src/components/rich-text/index.js

```diff
@@ -69,6 +69,21 @@ import { RemoveBrowserShortcuts } from './remove-browser-shortcuts';
 
 const { getSelection } = window;
 
+/**
+ * All inserting input types that would insert HTML into the DOM.
+ *
+ * @see  https://www.w3.org/TR/input-events-2/#interface-InputEvent-Attributes
+ *
+ * @type {Array}
+ */
+const INSERTION_INPUT_TYPES_TO_IGNORE = [
```

Comment:
FWIW `Set#has` (at least a native variant) performs much better than `Array#indexOf` and has arguably simpler semantics (alternatively considering `Array#includes`).

https://jsperf.com/array-indexof-vs-set-has

Reply:
Thanks! That makes sense